src/content/docs/hi/install/auto.mdx

```diff
@@ -138,3 +138,23 @@ Astro आपकी `src/` निर्देशिका में लाइव
 📚 **अपनी साइट परिनियोजित करें:** हमारी [परिनियोजन मार्गदर्शिका](/hi/guides/deploy/) में जानें कि वेब पर Astro परियोजना कैसे बनाएं और परिनियोजित करें।
 
 📚 **अपने कोडबेस को समझें:** हमारे [परियोजना संरचना मार्गदर्शिका](/hi/core-concepts/project-structure/) में Astro के परियोजना की संरचना के बारे में और जानें।
```

Comment:
```suggestion
```
I think these (and the one above) can be deleted now, because they've been replaced by the cards?

Reply:
Oh dang, my brain glossed over this!